polaris-react/src/components/DataTable/DataTable.tsx

```diff
@@ -425,46 +433,62 @@ class DataTableInner extends PureComponent<CombinedProps, DataTableState> {
     };
   };
 
-  private stickyHeaderScrolling = (event: Event) => {
+  private focusListener = (event: Event) => {
```

Comment:
This is used in order to allow for a keyboard use, who is tabbing through the heading buttons of the sticky header, to have the headings come into view if they are outside of view.